styleguide/screens/Patterns/Inputs/Inputs.js

```diff
@@ -0,0 +1,417 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import dedent from 'dedent';
+
+import Specimen from '../../../components/Specimen/Specimen';
+import { H, T, D, C, A } from '../../../components/Scaffold/Scaffold';
+
+import CheckboxGroup from '../../../../components/Form/CheckboxGroup/CheckboxGroup';
+import IconInput from '../../../../components/Form/IconInput/IconInput';
+import Input from '../../../../components/Form/Input/Input';
+import InputRange from '../../../../components/Form/InputRange/InputRange';
+import InputRangeWithHistogram
+  from '../../../../components/Form/InputRange/InputRangeWithHistogram';
+import Option from '../../../../components/Form/Select/Option';
+import RadioGroup from '../../../../components/Form/RadioGroup/RadioGroup';
+import Select from '../../../../components/Form/Select/Select';
+
+import shared from '../../../../styleguide/shared.css';
+import m from '../../../../globals/modifiers.css';
+
+export default class Inputs extends Component {
+  state = {
+    checkboxes: [],
+    radios: '',
+    input: '',
+    inputRange: 0,
+    multiInputRange: {
+      min: 0,
+      max: 100,
+    },
+    select: '',
+  };
+
+  handleCheckboxChange = (e, _, checkboxes) => {
```

Comment:
So as far as I can tell this is the first component we are documenting where we are including interactivity in the examples. Do you think we should add something similar in the `<Button />` examples? e.g. `onClick` show a modal

Reply:
@farazcsk It's an aside from this PR, but what benefit does the interactivity on the button add? IMHO, we should be adding as little as possible.

My thinking is that we should add interactivity where it provides value. The inputs being interactive is valuable as they have different characteristics when they have a value vs when they don't. We might add it for the documentation of say, the modal component, as its useful to see how it works when it is opened and closed. I'm not sure of the benefits of adding interactivity to the button component however, as is doesn't really change when you do interact with it. That said, if we were to add any, we should keep it as simple as possible, e.g., fire an alert rather than triggering a modal.

Reply:
@rdjpalmer I see what you are saying we don't want to overcomplicate things. Theoretically the different states of an input lets say a checkbox could documented without interactivity right? That is not what I am suggesting. I just think for constancy we could update the button documentation. That said it's not really relevant to this PR, it was just a thought l had looking over this. 😄